dashboard/app/models/pd/enrollment.rb

```diff
@@ -349,7 +333,7 @@ def set_application_id
     pd_match = ->(application) {application.try(:pd_workshop_id) == pd_workshop_id}
 
     application_id = nil
-    Pd::Application::ApplicationBase.where(user_id: user_id, application_year: APPLICATION_CURRENT_YEAR).each do |application|
+    Pd::Application::ApplicationBase.where(user_id: user_id, application_year: workshop.school_year).each do |application|
```

Comment:
oo good idea! i think this works, but it also relies on the assumption that the workshops start after 6/1 (school_year assumes 6/1 is the start of the school year), might be worth a comment just incase it causes a problem later on

Reply:
Good call!